File: src/etl/lambdas/etl_task_sql/local.js
import { lambda_handler } from './handler.js';
import { readFile } from 'fs/promises';

let event = JSON.parse(await readFile("localtest.json", "utf8"));

let context = {
  getRemainingTimeInMillis: () => 900_000
}
console.log( await lambda_handler(event, context));

File: src/etl/lambdas/update_etl_run_map/local.py
from handler import lambda_handler
import sys
import json

event = {}
context = {}

filename = 'localtest.json'
if len(sys.argv) == 2:
    filename = sys.argv[1]

f = open(filename)
event = json.load(f)
f.close()

json_object = json.dumps(lambda_handler(event, context), indent = 4) 
# print(json_object) 
